ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -520,7 +520,10 @@ private void SetupReactContext(ReactContext reactContext)
 
             var uiManagerModule = reactInstance.GetNativeModule<UIManagerModule>();
             var rootTag = uiManagerModule.AddMeasuredRootView(rootView);
-            rootView.TouchHandler = new TouchHandler(rootView);
+            rootView.TouchHandler = new TouchHandler(rootView)
```

Comment:
The TouchHandler is used to emit events so we need to inject the event dispatcher into it...